MAUI/DataGrid/performance.md

```diff
@@ -31,12 +31,28 @@ The default value of this property is `true`. If you want to improve loading and
 
 N> Download demo application from [GitHub](https://github.com/SyncfusionExamples/How-to-improve-scrolling-performance-in-.NET-MAUI-DataGrid).
 
+### DataGridImageColumn
```

Comment:
The LoadUIView property also applies to the image column, allowing images to be drawn directly in the `DataGridCell` instead of loading them through the `Image` control. To draw images in the `DataGridCell`, the source must be loaded using the [ImageSource.FromResource()](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.imagesource.fromresource?view=net-maui-9.0) method.